uvtools/dspec.py

```diff
@@ -849,7 +849,10 @@ def gen_window(window, N, alpha=0.5, edgecut_low=0, edgecut_hi=0, **kwargs):
         edgecut_hi : int, number of bins to consider as zero-padded at the high-side
             of the array, such that the window smoothly connects to zero.
         alpha : if window is 'tukey', this is its alpha parameter.
+        normalize : str, optional
+            set to 'rms' to divide by rms and 'mean' to divide by mean.
     """
+    assert normalization in ['none', 'mean', 'rms'], "normalization must be one of ['none', 'rms', 'mean']"
```

Comment:
If you set the default argument to be `None` above, you can change this to:
```
if normalization is not None:
    assert normalization in ["mean", "rms"], <blah blah blah>
```
This makes it a little clearer what the allowed "actual values" are.

Reply:
I like it. Have made this change.